Source/WebGPU/WebGPU/Queue.mm

```diff
@@ -693,7 +693,7 @@ static void invalidateCommandBuffers(Vector<std::reference_wrapper<CommandBuffer
     if (isCompressed && (widthMismatch || multipleOfBlockSize)) {
 
         auto maxY = std::max<size_t>(blockHeight, heightForMetal) / blockHeight;
-        auto newBytesPerImage = newBytesPerRow * std::max<size_t>(blockHeight, logicalSize.height / blockHeight);
+        auto newBytesPerImage = newBytesPerRow * std::max<size_t>(blockHeight, logicalSize.height / blockHeight + (logicalSize.height % blockHeight ? 1 : 0));
```

Comment:
once https://github.com/WebKit/WebKit/pull/35095 lands this will become:
```
auto checkedNewBytesPerImage = checkedProduct<uint32_t>(newBytesPerRow, std::max<size_t>(blockHeight, logicalSize.height / blockHeight), logicalSize.height % blockHeight ? 1 : 0);
```